go_metrics/metrics/graphite.py

```diff
@@ -141,12 +145,50 @@ def get(self, **kw):
         null_parser = null_parsers[params['nulls']]
         returnValue(self._parse_response((yield resp.json()), null_parser))
 
+    @inlineCallbacks
+    def fire(self, **kw):
+        config = self.backend.config
+        wc = WorkerCreator({
+            'hostname': config.amqp_hostname,
+            'port': config.amqp_port,
+            'username': config.amqp_username,
+            'password': config.amqp_password,
+            'vhost': config.amqp_vhost,
+            'specfile': config.amqp_spec,
+            })
+        worker = wc.create_worker_by_class(MetricWorker, {
+            'prefix': config.prefix})
+        yield worker._started_d
+        mm = worker.metric_manager
```

Comment:
We should probably only create the worker and metric manager once when the process starts, rather than connecting to AMQP each time a metric is fired.